### DISCUSSION TITLE
Connection Timeout

### DISCUSSION THREAD

**Discussion by marielb** [deepgram/community#539] [General help]
I was able to test out Deepgram a few days ago, but now I'm getting a timeout:

```
➜  ~ curl \
  --request POST \
  --header 'Authorization: Token <TOKEN>' \
  --header 'Content-Type: application/json' \
  --data '{"url":"https://static.deepgram.com/examples/nasa-spacewalk-interview.wav"}' \
  --url 'https://api.deepgram.com/v1/listen?punctuate=true'
curl: (28) Failed to connect to api.deepgram.com port 443: Operation timed out
```
My project ID Is `fe2aed9e-6939-4354-bc81-805c4c6fe6ef` and I've used <$1 worth of credits so far. I'm not sure what else to provide since I'm not able to connect.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @marielb, were you able to get this working? Your curl request works as-is for me (with my own API key), and returns results in about a second. Are you using any kind of proxy, firewall, or VPN? There's a good chance it's a networking issue from the origin where you're making the API request.

**Comment by marielb** [comment]
Yes I was able to get this working. The issue was that I live in Dominican Republic, but VPN-ing elsewhere fixed it.
